addons/website/views/snippets/s_cta_box.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_cta_box" name="CTA Box">
+    <section class="s_cta_box pt80 pb80">
+        <div class="container">
+            <div class="s_card o_card_img_horizontal flex-lg-row-reverse card o_cc o_cc4" data-vcss="001" data-vxml="001" style="--card-spacer-x: 64px;">
+            <figure class="o_card_img_wrapper ratio ratio-16x9 mb-0">
+                <img class="o_card_img card-img-top" src="/web/image/website.s_cta_box_default_image"/>
+            </figure>
+            <div class="card-body pt72 pb72">
```

Comment:
`pt72 pb72`: why not using `--card-spacer-y` like you did in L7? 🤔

Reply:
Wanted to allow the user to tweak the padding but looks like it doesn't work anyway 😅